htsohm/htsohm.py

```diff
@@ -262,7 +262,7 @@ def get_all_parent_ids(run_id, generation):
     result.close()
     return parent_ids
 
-def get_mutation_strength(run_id, generation, parent):
+def calculate_mutation_strength(run_id, generation, parent):
```

Comment:
we should discuss this, calculating the mutation strength calls calculate_percent_children_in_bin which is a class method of the material itself, furthermore this method requires the parent bin AND parent id to find children, so we need to query the parent_id anyway. i would have to pull calculate_percent_children_in_bin into htsohm/htsohm.py or move it somewhere else...